mitodl/open-discussions
# Prevent embedly from creating animated thumbnails

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #2210 

#### What's this PR do?
Adds `animate=false` to embedly requests

#### How should this be manually tested?
- Change the `image_src` attribute of one of your courses to an animated gif:
  ```python
   from course_catalog.models import Course
   from search.task_helpers import upsert_course
   course = Course.objects.get(id=5959)
   course.image_src = "https://ocw.mit.edu/courses/nuclear-engineering/22-06-engineering-of- nuclear-systems-fall-2010/22-06f10.gif"
   course.save()
   upsert_course(course)
   ```
- Search for that course at `../courses/search`.  The thumbnail in search results should not be animated.
- Click on that course card.  The thumbnail in the drawer should not be animated either.  Compare to https://discussions-rc.odl.mit.edu/courses/search?q=Engineering%20of%20Nuclear%20Systems

#### What GIF best describes this PR or how it makes you feel?
![source](https://user-images.githubusercontent.com/187676/66940604-3d008580-f013-11e9-9a73-15532bb22443.gif)
